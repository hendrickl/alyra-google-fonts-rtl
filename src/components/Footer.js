import React from "react"

const Footer = () => {
  return (
    <footer className="p-3 text-center bg-dark text-white">
      With Love & React.js for Alyra
    </footer>
  )
}

export default Footer
