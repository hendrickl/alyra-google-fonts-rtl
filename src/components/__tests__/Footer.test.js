import React from "react"
import Footer from "../Footer"
import { render, cleanup } from '@testing-library/react'

afterEach(cleanup)


test('<Footer />', () => {
  const {
    debug, container,
  } = render(<Footer />);

  expect(container.firstChild).toMatchSnapshot()
})
