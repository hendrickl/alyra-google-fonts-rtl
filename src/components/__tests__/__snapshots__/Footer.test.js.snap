// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> 1`] = `
<footer
  class="p-3 text-center bg-dark text-white"
>
  With Love & React.js for Alyra
</footer>
`;
